src/store/modules/job/actions.ts

```diff
@@ -228,10 +282,37 @@ const actions: ActionTree<JobState, RootState> = {
               'status': job?.statusId
             }
           })
+  
           if(payload.viewIndex && payload.viewIndex > 0){
             jobs = state.pending.list.concat(resp.data.docs);
           }
-          commit(types.JOB_PENDING_UPDATED, { jobs, total });
+          
+          const typeDefinedJobs: Job = jobs.map((response: any) => {
+            return { 
+              id: response.jobId,
+              name: response.jobName,
+              systemJobEnum: [
```

Comment:
Update this property to an object which will contain key-value pair for id, name, description and typeId

Reply:
okay sir!